tests/api/test_config.py

```diff
@@ -126,3 +129,69 @@ def test_check_initialized_bad_cluster_config(self) -> None:
         )
         with pytest.raises(ValueError, match="Missing server configuration"):
             config.check_initialized()
+
+
+class TestPyPIVersion:
```

Comment:
I couldn't find any unit test on `_PyPIVersion` so created a new one

Reply:
There is `cli/test_version_check.py` but it is a little different.

Reply:
moved there.